src/util/amount.rs

```diff
@@ -717,15 +927,40 @@ impl SignedAmount {
         SignedAmount::from_str_in(&value.to_string(), denom)
     }
 
+    /// Returns the absolute value as satoshis.
+    ///
+    /// This is the implementation of `unsigned_abs()` copied from `core` to support older MSRV.
+    fn as_sat_abs(self) -> u64 {
```

Comment:
nit: I think according to our latest naming discussions this should be `to_sat_abs`

Reply:
I nominate `to_`/`as_`/`into_` for category 'most annoying function naming convention to get correct' and apologize profusely for opening this particular can of worms.

Reply:
Clippy makes it reasonably easy though: https://rust-lang.github.io/rust-clippy/master/index.html#wrong_self_convention

Reply:
Oh sweeeet - I did not know about that lint. Cheers!

Reply:
Oh, yes, I made the name in line with the other methods without thinking.